### PR TITLE
[docs] edit documentation files related with tensor info @open sesame 10/04 14:33

### PR DESCRIPTION
**Pull request by niley7464**
This patch edits docs and comments related with tensor info.

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: Documentation/data-type-and-flow-control.md [hunk 1]
```diff
@@ -78,13 +78,13 @@ Typestrings = (string) Typestring
...
            | (string) Dimension, Dimensions
-Dimension = (string) [1-65535]:[1-65535]:[1-65535]:[1-65535]
+Dimension = (string) [[1-65535]:]{15}[1-65535]
 ```
 
 Note that dimension related values are only effective with static or sparse formats.
 With ```format=flexible```, types and dimensions are not effective.
 
-The buffer of ```other/tensors``` streams have multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With default configurations of Gstreamer 1.0, the maximum allowed number of memory chunks in a buffer is **16**. Thus, with such configurations of Gstreamer 1.0, ```other/tensors``` may include up to **16** ```other/tensor```.
+The buffer of ```other/tensors``` streams have multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With default configurations of Gstreamer 1.0, the maximum allowed number of memory chunks in a buffer is **16**. Thus, with such configurations of Gstreamer 1.0, ```other/tensors``` may include up to **16** static ```other/tensor``` and **240** extra ```other/tensor```.
```
Comment (wooksong) [inline Documentation/data-type-and-flow-control.md:87]:
```suggestion
The buffer of ```other/tensors``` streams has multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With the default configuration of GStreamer 1.0, the maximum allowed number of memory chunks in a buffer is **16**. Thus, ```other/tensors``` may include up to **16** static ```other/tensor``` and **240** extra ```other/tensor```.
```

Comment (wooksong) [inline Documentation/data-type-and-flow-control.md:87]:
```suggestion
The buffer of ```other/tensors``` streams has multiple memory chunks. Each memory chunk represents one single tensor in the buffer format of ```other/tensor```. With the default configuration of GStreamer 1.0, a GstBuffer can hold **16** memory chunks at most. Thus,  ```other/tensors``` may include up to **16** static ```other/tensor``` and **240** extra ```other/tensor```.
```

Comment (niley7464) [reply Documentation/data-type-and-flow-control.md]: Updated!
Thank you 👍

**Review by anyj0527 — APPROVED** [review]
LGTM👍 Thanks!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**